src/kOS.Safe/Compilation/KS/KSScript.cs

```diff
@@ -21,10 +21,6 @@ public override List<CodePart> Compile(string filePath, int startLineNum, string
         {
             List<CodePart> parts = null;
             
-            // make the code lowercase
-            scriptText = MakeLowerCase(scriptText);
-            scriptText = ReplaceIdentifiers(scriptText);
-            
             //if (contextId != "interpreter") parts = _cache.GetFromCache(scriptText);
 
             // if parts is null means the code doesn't exists in the cache
```

Comment:
Entirely unrelated to this commit.  If you look, that was already the case before I touched anything.